### ISSUE TITLE
Missing filed breaking the RackSpace Alamo post-installer

### ISSUE THREAD

**Issue by kieppie** [rcbops/chef-cookbooks#278]
Monitoring my Alamo v.2 ISO post-installer logs & it's failing to complete because it's unable to locate the appropriate resources online:
[Tue, 27 Nov 2012 21:39:59 -0500] [72] Generating chef-client keys
[Tue, 27 Nov 2012 21:40:02 -0500] [80] Downloading cookbooks
/root /root
error: The requested URL returned error: 403 while accessing http://github.com/rcbops/chef-cookbooks/info/refs
fatal: HTTP request failed
error: The requested URL returned error: 403 while accessing http://github.com/rcbops/chef-cookbooks/info/refs
fatal: HTTP request failed

This is because http://github.com/rcbops/chef-cookbooks/info/ no longer exists

**Comment by mancdaz** [comment]
closing here and tracking in https://github.com/rcbops/alamo-builder/issues/34

**Comment by kieppie** [comment]
@doublerr, @mancdaz - found the issue, thanks to the guidance of @Apsu.
Was the proxy config. I provided the URL to my local apt-cacher-ng apt repo proxy (since the prompt is nearly identical to the prompt in Ubuntu minimal) at install.
I cleared the proxy configs from my /opt/rpcs/rpcs.cfg, but kept my apt proxy in /etc/apt/apt.conf.d/00proxy, thereby keeping my apt proxy for my local repo intact, but not for general HTTP connections.
